src/client/ContentManagement.js

```diff
@@ -1023,18 +1024,21 @@ exports.PublishContentVersion = async function({objectId, versionHash, awaitComm
     throw Error(`Pending version hash mismatch on ${objectId}: expected ${objectHash}, currently ${pendingHash}`);
   }
 
+  let confirmCommitFound = false;
   if(awaitCommitConfirmation) {
     this.Log(`Awaiting commit confirmation for ${objectHash}`);
     const pollingInterval = this.ethClient.Provider().pollingInterval || 500;
+    const startTime = Date.now();
+    const duration = 2 * 60 * 1000; // 2 minutes
```

Comment:
We shouldn't use a global timeout here because object publishing times are proportional to the size of the change (a 6 hour high-res VOD object might take 10 min to publish).

I suggest we remove this timeout at this version and we could add a timeout parameter in the future if needed

Reply:
sure @elv-serban will make the changes.